src/DGtal/math/MultiStatistics.h

```diff
@@ -0,0 +1,461 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+#pragma once
+
+/**
+ * @file MultiStatistics.h
+ * @author Bertrand Kerautret (\c kerautre@loria.fr )
+ * LORIA (CNRS, UMR 7503), University of Nancy, France
+ *
+ * @author Jacques-Olivier Lachaud
+ * @date 2015/11/09
+ *
+ * Header file for module MultiStatistics.cpp
+ *
+ * This file is part of the DGtal library.
+ */
+
+#if defined(MultiStatistics_RECURSES)
+#error Recursive header files inclusion detected in MultiStatistics.h
+#else // defined(MultiStatistics_RECURSES)
+/** Prevents recursive inclusion of headers. */
+#define MultiStatistics_RECURSES
+
+#if !defined MultiStatistics_h
+/** Prevents repeated inclusion of headers. */
+#define MultiStatistics_h
+
+//////////////////////////////////////////////////////////////////////////////
+// Inclusions
+#include <iostream>
+#include "DGtal/base/Common.h"
+//////////////////////////////////////////////////////////////////////////////
+
+namespace DGtal
+{
+
+/////////////////////////////////////////////////////////////////////////////
+// class MultiStatistics
+/**
+ * Description of class 'MultiStatistics' <p> \brief Aim: This class
+ * stores a set of sample values for several variables and can then
+ * compute different statistics, like sample mean, sample variance,
+ * sample unbiased variance, etc.
+ *
```

Comment:
maybe you should add a note to make clear that a "terminale()" before accessing quantities